test/unit/specs/validators/email.spec.js

```diff
@@ -0,0 +1,48 @@
+import email from 'src/validators/email'
+
+describe('email validator', () => {
+  it('should not validate empty string', () => {
+    expect(email('')).to.be.false
+  })
+
+  it('should not validate numbers', () => {
+    expect(email('12345')).to.be.false
+  })
+
+  it('should not validate strings', () => {
+    expect(email('asdf12345')).to.be.false
+  })
+
+  it('should not validate space', () => {
+    expect(email(' ')).to.be.false
+  })
+
+  it('should not validate incomplete addresses', () => {
+    expect(email('someone@')).to.be.false
+    expect(email('someone@gmail')).to.be.false
+    expect(email('someone@gmail.')).to.be.false
+    expect(email('someone@gmail.c')).to.be.false
+    expect(email('gmail.com')).to.be.false
+    expect(email('@gmail.com')).to.be.false
+  })
+
+  it('should not validate addresses that contain unsupported characters', () => {
+    expect(email('someone@g~mail.com')).to.be.false
+    expect(email('some!one@gmail.com')).to.be.false
```

Comment:
Thanks for the feedback guys. I tried to make the regex as simple as possible.
What about this [one](http://emailregex.com/)?